src/shared/components/palette/dynamic-multi-select-list/multi-select-list.scss

```diff
@@ -0,0 +1,16 @@
+.multi-select-list-field-table {
```

Comment:
Should this be called `read-dynamic-multi-select-list-field.component.scss` since that's the only place it's used?